go/cmd/dolt/commands/remote_test.go

```diff
@@ -45,14 +45,14 @@ func TestGetAbsRemoteUrl(t *testing.T) {
 		{
 			"",
 			config.NewMapConfig(map[string]string{}),
-			"https://dolthub.com",
```

Comment:
Should this just reference `env. DefaultRemotesApiHost`?

Reply:
That's probably a better change for when this is changed from not being the beta site.